starter/README.md

```diff
@@ -3,21 +3,16 @@
 Servlet initial parameters may be passed via `@Component` annotation parameter `property`, e.g.
 
 ```java
-@Component(immediate = false, service = Servlet.class, property = {
-  HttpWhiteboardConstants.HTTP_WHITEBOARD_SERVLET_INIT_PARAM_PREFIX +
-  InitParameters.SERVLET_PARAMETER_COMPATIBILITY_MODE + "=false"}) 
+@Component(immediate = false, service = Servlet.class, property = { "some.property = someValue" })
 ```
 or via some annotation like `VaadinMode` which is available in the example.
 
-Same way can be used to set production mode via the initial parameters.
```

Comment:
Don't see a point to remove this if production mode is not used above as the main example.

Reply:
Reverted production mode to be only used in examples, so this line doesn't make sense now.